### PR TITLE
Develop010423

### PR DESCRIPTION
**Pull request by melnikalex1977**

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -1,6 +1,7 @@
 [flake8]
-ignore = E203, E266
+inline-quotes = "
+ignore = E203, E266, W503, ANN002, ANN003, ANN101, ANN102, ANN401, N807, N818
 max-line-length = 79
 max-complexity = 18
-select = B,C,E,F,W,T4,B9
+select = B,C,E,F,W,T4,B9,ANN,Q0,N8,VNE
```
Comment (ArsenPidhoretskyi) [inline .flake8:6]: Restore this changes

File: README.md [hunk 2]
```diff
@@ -21,11 +21,11 @@ Also `cache` should print `Getting from cache` when returns stored value and
 Example:
 ```python
 @cache
-def long_time_func(a, b, c):
+def long_time_func(a: int, b: int, c: int) -> int:
     return (a ** b ** c) % (a * c)
 
 @cache
-def long_time_func_2(n_tuple, power):
+def long_time_func_2(n_tuple: tuple, power: int) -> int:
```
Comment (ArsenPidhoretskyi) [inline README.md:28]: restore this changes

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]
Task is not done. Please be attentive. I do not see any changes. Also you must create merge into master

**Review by melnikalex1977 — COMMENTED** [review]
Changes 010423

**Review by melnikalex1977 — COMMENTED** [review]
changes3

**Comment by Julia15031988** [comment]
hello

**Pull request closed without merging**